plot-api/src/commonMain/kotlin/jetbrains/letsPlot/GlobalSettings.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot
```

Comment:
Let's move GlobalSettings to the "jetbrains.letsPlot.intern.settings" package.

Reply:
Done